docs/release-checklist.rst

```diff
@@ -0,0 +1,155 @@
+
+=================
+Release Checklist
+=================
+
+meejah produced this list while making the 1.15.0 release. Many of the
+things in the `how_to_make_a_tahoe-lafs_release.org` document aren't
+relevant anymore. However, until we are sure that a "new release list"
+is correct and works, I don't just want to completely revise it.
+
+A major difference here is splitting into things that "any
+contributer" can do and things that contributers with possibly more
+access need to do.
+
+So, follow *this* list for 1.16.0 and if it seems complete, we should
+delete the older checklist after that.
+
+
+Any Contributor
+---------------
+
+Anyone who can create normal PRs should be able to complete this
+portion of the release process.
+
+
+Prepare for the Release
+```````````````````````
+
+The `master` branch should always be releasable. However, it is worth
+asking on appropriate channels (IRC, the mailing-list, Nuts and Bolts
+meetings) whether there are interesting changes that should be
+included (or NOT included) etc.
+
+- Create a ticket for the release in Trac
+- Ticket number needed in next section
+
+
+Create Branch and Apply Updates
+```````````````````````````````
+
+- Create a branch for release-candidates (e.g. `release-1.15.0.rc0`)
+- run `tox -e news` to produce a new NEWS.txt file (this does a commit)
+- create the news for the release
+  - newsfragments/<ticket number>.minor
+  - commit it
+- manually fix NEWS.txt
+  - proper title for lastest release (instead of "Release ...post1432")
+  - double-check date
+  - spot-check the release notes (these come from the newsfragments
+    files though so don't do heavy editing)
```

Comment:
I guess because whomever wrote and reviewed a particular change presumably thought about what to write about it and someone else changing it all "as" the release happens seems .. bad?